.circleci/config.yml

```diff
@@ -326,6 +326,9 @@ jobs:
       name: docker
       py-version: << parameters.py-version >>
 
+    environment:
+      RAIDEN_TESTS_SYNAPSE_LOGS_DIR: /tmp/synapse-logs
```

Comment:
Are these stores as artifacts in circle?

Reply:
No, In general we shouldnt need them. But if anything happens the logs are there if one would ssh into the job.